### ISSUE TITLE
Qwave Raspbian-003.img boot problem

### ISSUE THREAD

**Issue by khan-yousaf** [QWaveSystems/QwaveSys-Raspberry-Pi-Package#1]
While booting my Raspberry pi 3 with your 003.img, the system hangs and do not proceed further. Given below is a screen shot of my system..

![20160924_094350](https://cloud.githubusercontent.com/assets/8445997/18806206/c5b2e040-823d-11e6-8ce1-e9c0c99d7810.jpg)

**Comment by pamornthep** [comment]
The image from original (AWS server) link seem like it's has corrupted, 

Please download the same image from this alternative link.

http://iosys.link/Qwave-Raspbian_qwavesys-rpi-003.img

I will update the WiKi for the new link accordingly.

Amornthep

**Comment by ahmedzahid** [comment]
I also faced the similar issue and I'm re downloading now using alternative link. Thanks

**Comment by ahmedzahid** [comment]
I've used the image downloaded from the alternative link. On very first time, I was able to connect to see the login screen of raspberry Pi upon connecting with remote desktop. However, suddenly the connection failed and all the subsequent attempts to connect with the remote desktop were unsuccessful. Then I unplugged it and plugged with the HDMI, I saw some error message somewhat like the above on the screen. Moving forward I repeated the steps i.e formatted the SD card again and wrote the image file. This time the boot is stuck at black blank screen. I've shot a video of the boot sequence. I can share that too. What could be the possible reason? Any idea or way around?
